### PR TITLE
Aml 3610 move sfa das caches

### PR DESCRIPTION
**Pull request by tom-gough**

### CODE REVIEW

File: src/SFA.DAS.Account.Api.Client.UnitTests/SFA.DAS.EAS.Account.Api.Client.UnitTests.csproj [hunk 1]
```diff
@@ -53,8 +53,9 @@
     <Reference Include="Moq, Version=4.5.28.0, Culture=neutral, PublicKeyToken=69f491c39445e920, processorArchitecture=MSIL">
       <HintPath>..\packages\Moq.4.5.28\lib\net45\Moq.dll</HintPath>
     </Reference>
-    <Reference Include="Newtonsoft.Json, Version=10.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed, processorArchitecture=MSIL">
-      <HintPath>..\packages\Newtonsoft.Json.10.0.3\lib\net45\Newtonsoft.Json.dll</HintPath>
+    <Reference Include="Newtonsoft.Json, Version=11.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed, processorArchitecture=MSIL">
+      <SpecificVersion>False</SpecificVersion>
+      <HintPath>..\..\..\..\..\Users\Tom\.nuget\packages\newtonsoft.json\11.0.2\lib\net45\Newtonsoft.Json.dll</HintPath>
```
Comment (MichaelYoung1981) [inline src/SFA.DAS.Account.Api.Client.UnitTests/SFA.DAS.EAS.Account.Api.Client.UnitTests.csproj]: Probably remove the hint path.

File: src/SFA.DAS.EAS.Api/Web.config [hunk 2]
```diff
@@ -320,6 +320,10 @@
         <assemblyIdentity name="Microsoft.IdentityModel.Clients.ActiveDirectory" publicKeyToken="31bf3856ad364e35" culture="neutral" />
         <bindingRedirect oldVersion="0.0.0.0-5.0.5.0" newVersion="5.0.5.0" />
       </dependentAssembly>
+      <dependentAssembly>
```
Comment (MichaelYoung1981) [inline src/SFA.DAS.EAS.Api/Web.config]: Why is this needed?

File: src/SFA.DAS.Account.Api.Client.UnitTests/SFA.DAS.EAS.Account.Api.Client.UnitTests.csproj (hunk 1 above)
Comment (tom-gough) [reply src/SFA.DAS.Account.Api.Client.UnitTests/SFA.DAS.EAS.Account.Api.Client.UnitTests.csproj]: done

File: src/SFA.DAS.EAS.Api/Web.config (hunk 2 above)
Comment (tom-gough) [reply src/SFA.DAS.EAS.Api/Web.config]: it's not, removed
